### ISSUE TITLE
Is L1 loss function correct?

### ISSUE THREAD

**Issue by OnlyBelter** [KevinMenden/scaden#37]
I saw L1 loss function in your code 
`loss = tf.reduce_mean(input_tensor=tf.math.square(logits-targets))`
on the line of https://github.com/KevinMenden/scaden/blob/master/scaden/model/scaden.py#L67

Is it correct?

In my understand, it should be the sum of absolute value, right?

This is the defination of L1 norm:
https://mathworld.wolfram.com/L1-Norm.html

**Comment by KevinMenden** [comment]
Hi,

thanks for pointing that out - this is indeed the L2 loss in the current version.
I have to check when this was changed, looks like a mistake ... however during testing we could not detect significant differences between L1 and L2 loss.

Again, thanks for pointing that out, that's a 'bug'! I will leave it at L2 for now (as it doesn't make a difference), but have to change the documentation obviously :-)

**Comment by OnlyBelter** [comment]
You are welcome!
